### PR TITLE
[JA] Update `Ranking_criteria/osu!taiko`

### PR DESCRIPTION
**Pull request by mangomizer**
Update JP Taiko RC to match EN, translation courtesy of TKS and KSHR!

### CODE REVIEW

File: wiki/Ranking_Criteria/osu!taiko/ja.md [hunk 1]
```diff
@@ -1,45 +1,183 @@
...
+- **スライダーの目盛(Slider Tick Rate)は曲のリズムに応じて設定する必要があります。** 多くの場合は1に設定しますが、1/3(12分音符)を主に使用するような曲の場合は、3に設定して、スライダーの目盛りも1/3のリズムで配置されるようにします。
+- **どのリズムやどの音(楽器やボーカルなど)を優先すべきかわかりづらい曲の場合、複数の要素をフォローすることは避けましょう。** 何のリズムや音に焦点が当てられているかプレイヤーが理解できなければいけません。
+- **低音量、無音のスピナーを使用する場合は、曲の音量が小さい部分で使用してください。** それ以外の多くの場面では、スピナーの音が聞こえる音量設定をおすすめします。
+- **BPMが変わる曲では音符が流れる速度をほぼ一定に保つために、SliderVelocityを頻繁に変更しても構いません。** そうすることで、音符間のタイミングのずれが予測しやすくなったり、音符が重なってしまうことを回避して、ゲーム性を高めることが可能です。
+- **スピナーによってプレイフィールド上の音符を妨害しないようにしてください。** スピナーは画面の大部分を占めているため、次の音符との距離が近すぎると、誤認識が発生する可能性があります。原則、スピナーと次の音符までに1/2の間隔を空けます。
+- **カスタムヒットサウンドを使用する場合、打楽器に関連したものを使用してください。** 重い/低い音はドン、軽い/高い音はカツで使用してください。
+
+## 難易度別(Difficulty-specific)
+
+難易度別のルール、ガイドラインはそれぞれの難易度ごとに適用されるもので、すべてのTaiko譜面全体に適用されるものではありません。リズムに関するルール、ガイドラインは「4/4拍子、BPM180程度の譜面」を基準に適用されますが、楽曲が極端に遅い、速い場合にはRanking CriteriaのScaling BPMにあるように、許容範囲が一部異なる場合があります。
```
Comment (TPGPL) [inline wiki/Ranking_Criteria/osu!taiko/ja.md:76]: missing a link to RC/Scaling_BPM

**Review by TPGPL — CHANGES_REQUESTED** [review]

File: wiki/Ranking_Criteria/osu!taiko/ja.md [hunk 2]
```diff
@@ -1,45 +1,183 @@
----
-outdated: true
----
-
 # osu!taiko Ranking Criteria
 
-For the record, these include the recent changes made [from the discussion forum](https://osu.ppy.sh/community/forums/87). Any rule that is being discussed is listed here as it used to be and will be updated once the discussion reaches an agreement.
+osu!taiko Ranking Criteria(osu!taiko譜面のランク基準)はTaiko譜面を作成する際に従わなければならないルール、ガイドラインです。Taiko譜面をRankedさせるためにはこれらのルールに従う必要があります。いかなる場合でもルールに従わなければなりませんが、ガイドラインについては例外的な状況下において無視することが出来ます。ただし、その場合は何故ガイドラインを無視しないとTaiko譜面全体の品質に影響を及ぼすのか、具体的な説明が必要となります。
+
+これらのルールやガイドラインはRanking Criteriaのフォーラムで議論され決定されたものです。もし新しいルールやガイドラインが提案、合意に至った場合は本ページの内容も適宜更新されます。
```
[the remark this answers is not in the document]
Comment (mangomizer) [reply wiki/Ranking_Criteria/osu!taiko/ja.md]: Added

File: wiki/Ranking_Criteria/osu!taiko/ja.md (hunk 1 above)
Comment (mangomizer) [reply wiki/Ranking_Criteria/osu!taiko/ja.md]: Added

File: wiki/Ranking_Criteria/osu!taiko/ja.md [hunk 3]
```diff
@@ -1,45 +1,183 @@
----
-outdated: true
----
-
 # osu!taiko Ranking Criteria
 
-For the record, these include the recent changes made [from the discussion forum](https://osu.ppy.sh/community/forums/87). Any rule that is being discussed is listed here as it used to be and will be updated once the discussion reaches an agreement.
+osu!taiko Ranking Criteria(osu!taiko譜面のランク基準)はTaiko譜面を作成する際に従わなければならないルール、ガイドラインです。Taiko譜面をRankedさせるためにはこれらのルールに従う必要があります。いかなる場合でもルールに従わなければなりませんが、ガイドラインについては例外的な状況下において無視することが出来ます。ただし、その場合は何故ガイドラインを無視しないとTaiko譜面全体の品質に影響を及ぼすのか、具体的な説明が必要となります。
```
[the remark this answers is not in the document]
Comment (mangomizer) [reply wiki/Ranking_Criteria/osu!taiko/ja.md:3]: I'll keep it unchanged (it's pretty unambiguous) + I don't want to question the TL work done by others.

**Review by TPGPL — APPROVED** [review]

**Pull request merged**